tools/toitdoc/src/shadow.toit

```diff
@@ -439,10 +435,11 @@ compute-override-phase-positional super-iter/NamedIterator -> bool
     for i := from; i <= to; i++:
       if arities[i] <= 0:
         // Either another old overrider, or not overridden yet.
-        class-depth := -hack-class-depths[hack-holders[old-member.as-toit-member]]
-        if class-depth < arities[i]:
+        // Note that we use the negative of the inheritance order as discussed above.
+        inheritance-order := -old-member.inheritance-order
```

Comment:
Maybe give this a different name? Once negated, it is no longer the inheritance order.

Reply:
done.